libs/perun/components/src/lib/perun-footer/perun-footer.component.ts

```diff
@@ -55,14 +56,19 @@ export class PerunFooterComponent implements OnInit {
   constructor(
     private storeService: StoreService,
     private translateService: TranslateService,
+    private utilsService: UtilsService,
     private dialog: MatDialog
   ) {}
 
   ngOnInit(): void {
     this.translateService.onLangChange.subscribe((lang) => {
       this.language = lang.lang;
     });
-    this.version = require('../../../../../../package.json').version as string;
```

Comment:
It's a bit confusing that now we display the version of back-end, however the link in the footer still leads to GUI releases. I would probably update the version in package.json and then display both the version of GUI and BE.

Reply:
Yeah, you're right about the links. The following commit addresses this; the footer will now display both information (i.e. GUI and BE version) and their respective links